frontend/benefit/handler/src/components/newApplication/formContent/companySection/deMinimisAid/utils/validation.ts

```diff
@@ -0,0 +1,46 @@
+import { DE_MINIMIS_AID_GRANTED_AT_MAX_DATE } from 'benefit/handler/constants';
+import {
+  DE_MINIMIS_AID_KEYS,
+  VALIDATION_MESSAGE_KEYS,
+} from 'benefit-shared/constants';
+import { DeMinimisAid } from 'benefit-shared/types/application';
+import isFuture from 'date-fns/isFuture';
+import { TFunction } from 'next-i18next';
+import { convertToUIDateFormat, parseDate } from 'shared/utils/date.utils';
+import { getNumberValue } from 'shared/utils/string.utils';
+import * as Yup from 'yup';
+
+export const getValidationSchema = (t: TFunction): Yup.SchemaOf<DeMinimisAid> =>
+  Yup.object().shape({
+    [DE_MINIMIS_AID_KEYS.GRANTER]: Yup.string()
+      .required(VALIDATION_MESSAGE_KEYS.REQUIRED)
+      .max(64, (param) => ({
+        max: param.max,
+        key: VALIDATION_MESSAGE_KEYS.STRING_MAX,
+      })),
+    [DE_MINIMIS_AID_KEYS.AMOUNT]: Yup.number()
+      .transform((_value, originalValue) => getNumberValue(originalValue))
+      .required(VALIDATION_MESSAGE_KEYS.REQUIRED)
+      .typeError(VALIDATION_MESSAGE_KEYS.NUMBER_INVALID)
+      .min(0, (param) => ({
+        min: param.min,
+        key: VALIDATION_MESSAGE_KEYS.NUMBER_MIN,
+      })),
+    [DE_MINIMIS_AID_KEYS.GRANTED_AT]: Yup.string()
```

Comment:
Granted at not required in yup but is checked to be filled to get the submit button enabled 🤔